docs/public/installation/zonemaster-backend.md

```diff
@@ -210,6 +210,12 @@ sudo apt install jq libclass-method-modifiers-perl libconfig-inifiles-perl libdb
 > **Note**: libio-stringy-perl is listed here even though it's not a direct
 > dependency. It's an undeclared dependency of libconfig-inifiles-perl.
 
+For Ubuntu 20.04 only, install JSON::Validator from CPAN as the one in the system packages repository is too old:
+
+```sh
+sudo cpanm --notest JSON::Validator

```

Comment:
1. Should all instances of "cpanm" be with "--notest", or why just this one?
2. In the installation document we added the following text. Should it be here too?

>If you have any issue with installation, and installed with cpanm, redo the installation above but without the --notest option. Installation will take longer time.